### PR TITLE
Adding EJ Screen Areas of Concern

### PR DESCRIPTION
**Pull request by VincentLaUSDS**

### CODE REVIEW

File: data/data-pipeline/data_pipeline/etl/sources/ejscreen_areas_of_concern/etl.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+logger = get_module_logger(__name__)
+
+
+class EJScreenAreasOfConcernETL(ExtractTransformLoad):
+    def __init__(self):
+        self.OUTPUT_PATH = self.DATA_PATH / "dataset" / "ejscreen_areas_of_concern"
+
+        # TO DO: Load from actual source; the issue is that this dataset is not public for now
+        self.LOCAL_CSV_PATH = self.DATA_PATH / "local"
+        self.GEOID_CBG_FIELD_NAME = "GEOID10_CBG"
```
Comment (VincentLaUSDS) [inline data/data-pipeline/data_pipeline/etl/sources/ejscreen_areas_of_concern/etl.py:15]: Should just use `self.GEOID_FIELD_NAME` I think

Comment (lucasmbrown-usds) [reply data/data-pipeline/data_pipeline/etl/sources/ejscreen_areas_of_concern/etl.py:15]: Makes sense!

**Review by lucasmbrown-usds — APPROVED** [review]

**Comment by lucasmbrown-usds** [comment]
Actually, merging is blocked on this because the file is only being hosted locally for now.

**Comment by esfoobar-usds** [comment]
Do you need any help uploading the file to S3?

**Comment by esfoobar-usds** [comment]
Do you need any help uploading the file to S3?

**Comment by VincentLaUSDS** [comment]
@esfoobar-usds -- I think so! Although I now recall I thought the issue with keeping it local was because EPA did not want those data sets public yet?

**Comment by lucasmbrown-usds** [comment]
Closed in favor of https://github.com/usds/justice40-tool/pull/737, which has a cleaner rebased version without merge conflicts. I had to cherrypick because I'm not sure what happened but I think @VincentLaUSDS and I were doing different merge styles (I had been rebasing the topic branch with main, I think he merged main into the topic branch), and resolving the conflicts was a nightmare.

**Pull request closed without merging**